### DISCUSSION TITLE
Starting Details for Administrator Accounts & Sessions Team

### DISCUSSION THREAD

**Discussion by evnd04** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#7] [Informational]
Admin Interface Team responsibilities:

 Similar to User Interface, the Admin (Security Guards) accounts are a more elaborate version of the User Interface. The differences are:
> Able to access reports
> Able to receive specific alerts (SOS, Panic Alert, Immediate help)

The log in features that users share are as follows:
> Successful login
> Authentication, session management, and error responses.
> Buttons, input fields, and error messages.
> Incorrect password handling
> Account lockout after multiple failed attempts.
> Use UPRM email to ensure they’re UPRM guards and not outsiders
> Account Creation (Validate e-mail)
> Admin’s profile

Provide name, username, institutional email to be a part of this group, add (team leader) if interested in being a team leader.

Team Leader: @yadrielsamir  

@RaquelQuinones 
@JorgeC08
@javier-rivera8 
@Rodal1
@Chosen-Juan1 

This team is currently: *OPEN* and missing 4 developers and a Team Leader

**Please note:** Two of the members in this team need to take on the responsibility of handling both documentation (while reporting to @lex939 on this) and software development for admin interface. These two members will split their time 50-50 between documentation and development, so each will do a bit less of each task to collectively meet the same total of tasks per milestone. Please come to an agreement as a team on which people will be filling these roles.

**Comment by JorgeC08** [comment]
Name: Jorge Colon
Github: JorgeC08
E-mail: jorge.colon27@upr.edu

**Comment by yadrielsamir** [comment]
Yadriel Calderón 
yadrielsamir
yadriel.calderon@upr.edu

**Comment by javier-rivera8** [comment]
Javier Rivera Rodriguez
Github: javier-rivera8
E-mail: javier.rivera97@upr.edu

**Comment by Rodal1** [comment]
Robert Rodriguez
rodal1
robert.rodriguez11@upr.edu

**Comment by Chosen-Juan1** [comment]
Juan D. Guadalupe Rosado
Github: Chosen-Juan1
Email: [juan.guadalupe@upr.edu](mailto:juan.guadalupe@upr.edu)

**Comment by YanehRuiz** [comment]
Hello everyone. This team consists of six people. Therefore I'd like to ask if any of you would like to step up as a team leader.

@RaquelQuinones 
@JorgeC08
@yadrielsamir 
@javier-rivera8 
@Rodal1
@Chosen-Juan1

**Reply by RaquelQuinones** [reply]
I do not want to be team leader.
